Cloud Pak for Data/WML/notebooks/model risk management/MRM E2E with Openpages on CP4D.ipynb

```diff
@@ -222,11 +222,13 @@
    "source": [
     "!rm -rf /home/spark/shared/user-libs/python3.6*\n",
     "\n",
-    "!pip install --upgrade pyspark==3.0.3  --no-cache | tail -n 1\n",
-    "!pip install numpy==1.20.1 --no-cache | tail -n 1\n",
-    "!pip install scikit-learn==0.23.1 --no-cache | tail -n 1\n",
-    "!pip install --upgrade pandas==1.2.3 --no-cache | tail -n 1\n",
-    "!pip install --upgrade SciPy --no-cache | tail -n 1\n",
+    "#If you are running this notebook in non IBM Watson Studio env then uncomment the below pip statements and run it\n",
+    "#!pip install --upgrade pyspark==3.0.3  --no-cache | tail -n 1\n",
```

Comment:
We should install pyspark 3.2.x because that's what is given below spark-mllib_3.2